_includes/main.scss

```diff
@@ -451,7 +451,38 @@ h3 {
 
 form {
   border-radius: 4px;
-  padding: 1rem;
+  padding: 0 1.5rem 1rem 1.5rem;
+  margin: auto;
+
+  label {
+    margin: 0;
+    width: 100%;
+
+    div, input::placeholder {
+      padding-top: 1rem;
+      padding-bottom: 2px;
+      margin-left: 5px;
+    }
+
+    input::placeholder {
+      text-align: center;
```

Comment:
I'd keep the left-aligned, having them float far away to the right throws me off. It's also odd that the placeholder alignment is different from real form values

If we insist on centering this stuff, we should also consider that the `select[name=nearby-chapter]` is currently left-aligned